### DISCUSSION TITLE
[Feature] Update Camera Make/Model

### DISCUSSION THREAD

**Discussion by PrivateButts** [immich-app/immich#10311] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I've just started poking around with Immich and a feature that I'd love is the ability to update the camera make and model on a photo in a similar way to the location and date. My use case here is for film scans and other non-direct digital photos, which will typically have the EXIF information of the device used to make the scan rather than the camera itself.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by AngelaDMerkel** [comment]
The best way to do this is just to use exiftool on your scans, and if you don't need precise data, it can be done very painlessly:
```
exiftool -Make="Yashica" -Model="Mat-124 G" -LensMake="Yashinon" -LensModel="Copal-SV 80mm f3.5" -Lens="Yashinon Copal-SV 80mm f3.5" -DateTime="2023:09:24 14:28" -DateTimeOriginal="2023:09:24 14:28" -ShutterSpeedValue="1/500" -ExposureTime="1/500" -ApertureValue="8" -FNumber="8" -FocalLength="80" -ISO="400" -LightSource="Fine Weather" *.tif
```
If you only have one camera this whole process can be done in seconds for all of your images.

**Reply by PrivateButts** [reply]
exiftool is nice, but I would love to see an GUI option within the app as well, as my immich server will be used by folks who aren't as comfortable with the command line as others. It'd also be handy to do the corrections with the bulk editor as part of the flow of correcting location and date, which is already in immich.

**Reply by AngelaDMerkel** [reply]
This is a really good point, but how many users are uploading scanned images to immich? I am not objecting to it as a feature, but I'd be curious if your users care that much. None of my users care about the date or camera for their scanned photos and just let the hang out wherever in the timeline.

**Reply by PrivateButts** [reply]
My wife and I reverted mostly to film over the last couple of years because we enjoy the little rituals it involves. Basically, anything that isn't impromptu dog or rabbit photos are shot on whatever little weird film thing I've fixed up that month, so we do end up with a bunch of scans from a variety of film cameras. I understand that this use case isn't going to be the most common, but it would be nice, especially since camera make/model is a filter that you can use when searching.

**Reply by AngelaDMerkel** [reply]
I have something like 15K individual film scans taken over the years, and a lot of this was made easier by using [Exifnotes](https://github.com/tommi1hirvonen/ExifNotes) which automatically generates the exiftool commands based on the details you note down. It includes information including location and lens, or aperture if you choose to use it which can be enormously helpful. It's an excellent help for determining some of the characteristics of lenses are different settings in different circumstances. It's an exceptionally helpful tool.
